.travis.yml

```diff
@@ -5,11 +5,13 @@ git:
   depth: 9999999
 jdk:
 - oraclejdk8
+- openjdk8
```

Comment:
It means we run our tests twice.  It doubles the compute but only increases the wallclock time slightly due to the fact that we have twice as many chances for a weird slow outlier to happen.  We could just replace oracle with openjdk if we don't want to consume more compute.